rwslib/builders/metadata.py

```diff
@@ -389,26 +438,31 @@ class StudyEventRef(ODMElement):
     The list of :class:`StudyEventRef` identifies the types of study events that are allowed to occur within the study.
     The :class:`StudyEventRef` within a :class:`Protocol` must not have duplicate StudyEventOIDs nor
     duplicate OrderNumbers.
+
+    .. note::
+        In Rave the `OrderNumber` attribute is not reported for the **SUBJECT** Form, for this reason the order_number
+        attribute is not mandatory and will default to None
+
     """
 
-    def __init__(self, oid, order_number, mandatory):
+    def __init__(self, oid, order_number=None, mandatory=False):
         """
         :param oid: :class:`StudyEventDef` OID
         :type oid: str
         :param order_number: OrderNumber for the :class:`StudyEventRef` within the :class:`Study`
-        :type order_number: int
+        :type order_number: Union(int, None)
         :param mandatory: Is this StudyEventDef Mandatory? (True|False)
         :type mandatory: bool
         """
         self.oid = oid
-        self.order_number = order_number
+        self._order_number = order_number
         self.mandatory = mandatory
 
     def build(self, builder):
         """Build XML by appending to builder"""
-        params = dict(StudyEventOID=self.oid,
-                      OrderNumber=str(self.order_number),
-                      Mandatory=bool_to_yes_no(self.mandatory))
+        params = dict(StudyEventOID=self.oid, Mandatory=bool_to_yes_no(self.mandatory))
+        if self._order_number:
```

Comment:
0 is an int but would be dropped here. I am not sure that 0 is allowed as an Ordinal anyway, just noting it.

Reply:
Great point, updated!